packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx

```diff
@@ -44,25 +45,25 @@ export const CollectibleList = memo<CollectibleListProps>(({ selectedNetwork })
     const account = useAccount()
     const { Asset } = useWeb3PluginState()
     const network = useNetworkDescriptor()
-    const [loadingSize, setLoadingSize] = useState<number>()
-    const [loadingCollectible, setLoadingCollectible] = useState(true)
+    const [loadingSize, setLoadingSize] = useState(0)
```

Comment:
why initial value not is NaN?

```suggestion
    const [loadingSize, setLoadingSize] = useState(Number.NaN)
```

Reply:
Initialize with `0` is OK.